lib/services/user_service.dart

```diff
@@ -7,9 +7,12 @@ import 'package:nowu/services/auth.dart';
 class UserService {
   // TODO We hardly actually need the user now we have the auth token
   UserProfile? _currentUser = null;
+
   UserProfile? get currentUser => _currentUser;
   Logger _logger = Logger('UserService');
 
+  String? prefilledName;
```

Comment:
I'm tempted to say we should keep this in the auth service given the method getCurrentUserName lives there. What do you think?